sdk/core/core-http/package.json

```diff
@@ -82,7 +82,7 @@
     "integration-test:node": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint package.json api-extractor.json src test --ext .ts -f html -o core-http-lintReport.html || exit 0",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts -f html -o core-http-lintReport.html",
```

Comment:
Feel free to let me know if I should revert this - it was easier for me to see the `lint` failure until I cleaned up the errors and then I figured it might be good to prevent future errors?

Reply:
I think this was done to prevent lint from failing CI at one point, though maybe we don't need it anymore? /cc @willmtemple